tests/integration_tests/test_uv_functionality.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for UV functionality and integration validation."""
+
+from __future__ import annotations
+
+import os
+import subprocess
+from pathlib import Path
+
+import pytest
+
+
+def test_uv_available() -> None:
+    """Test that UV is available on the command line."""
+    # Use poetry's virtual environment where UV is installed
+    result = subprocess.run(
+        ["poetry", "run", "uv", "--version"],
```

Comment:
I see I missed this problem in the original tests. Stop everything else until this is working without a poetry ref. You need to be able to call UV without poetry's help. The venv we are creating probably doesn't need UV installed. Instead, you can use the UV installation from our PyAirbyte environment.